crates/mysten-util-mem/Cargo.toml

```diff
@@ -15,7 +15,7 @@ lru = { version = "0.7", optional = true }
 hashbrown = { version = "0.12", optional = true }
 mysten-util-mem-derive = { path = "../mysten-util-mem-derive", version = "0.1" }
 impl-trait-for-tuples = "0.2.0"
-fastcrypto = { git = "https://github.com/MystenLabs/fastcrypto", rev = "c022a2ae23ca7cc2778293fd3b1db42e8cd02d3b", features = ["copy_key"] }
+fastcrypto = { git = "https://github.com/MystenLabs/fastcrypto", version = "0.1", features = ["copy_key"] }
```

Comment:
If fastcrypto has been published then maybe drop the git ref?

Reply:
Sent a followup PR, sorry for the churn